sedaro/src/sedaro/results/agent.py

```diff
@@ -65,7 +66,7 @@ def block(self, id_: str) -> SedaroBlockResult:
                 block_data[module]['time'] = self.__series[module]['time']
                 block_data[module]['series'] = self.__series[module]['series'][id_]
         block_structure = self.__block_structures[id_] if id_ != 'root' else id_
-        return SedaroBlockResult(block_structure, block_data)
+        return SedaroBlockResult(block_structure, block_data, self.__axis)
```

Comment:
What is the motivation for threading `axis` through to all these `Results` tiers/classes?  Why not just instantiate the top level `Results` object with data in the intended axis order and allow everything else to assume the axis order is a certain way?